pygmt/helpers/utils.py

```diff
@@ -124,64 +130,53 @@ def data_kind(data=None, x=None, y=None, z=None, required_z=False, required_data
     * 1-D arrays x and y (and z, optionally)
```

Comment:
Delete this line about x/y/z?

```suggestion
```

Reply:
Updated the docstring at b575591ace1374b5bdc547f94fcf1262aacf247d. Currently, any unrecognized data type is taken as a `matrix` type. It's not ideal, so I plan to refactor the data_kind function later, and will also polish the docstring when refactoring.